deploy/deploy-wmagent.sh

```diff
@@ -372,13 +372,17 @@ echo "Done!" && echo
 echo "*** Setting up US opportunistic resources ***"
 if [[ "$HOSTNAME" == *fnal.gov ]]; then
   sed -i "s+forceSiteDown = \[\]+forceSiteDown = \[$FORCEDOWN\]+" $MANAGE_DIR/config.py
-  cd $BASE_DIR
-  wget -nv https://raw.githubusercontent.com/dmwm/WMCore/master/deploy/addUSOpportunistic.sh -O addUSOpportunistic.sh
-  chmod +x addUSOpportunistic.sh
-  $BASE_DIR/addUSOpportunistic.sh
-  cd -
+  local PEND_JOBS=2000
```

Comment:
I think the more variables - or constants - we have in this script, the more complicated it becomes. IMO, the most important is to get these resources into the agent database; then we should make sure that whoever is managing such opportunistic resources, properly communicate it with the SS team and define the correct number of slots to be used. This way, once the components are started, the agent would automatically adapt the number of slots for each of those resources, *provided* they are present in the SSB site metrics that agents consume.

I should definitely avoid that duplication of (re-)defining the slots variable though and will update it soon.

Reply:
if we want to take that strategy it is fine too.

Reply:
Yes, I think having reasonable default values should be good enough. In the end, we have two problems:
* adding resources to the agent
* setting the proper number of slots

for the latter, I see no blocker on why it shouldn't be like this since the day 1 (and there is no development to be done on our side).

Thanks for your review Todor.